datalake/src/test/java/com/sequenceiq/datalake/service/sdx/database/AwsDatabaseServerParameterSetterTest.java

```diff
@@ -46,18 +46,29 @@ public void testHAServer() {
 
     @Test
     public void testNonHAServer() {
-        underTest.setParameters(request, SdxDatabaseAvailabilityType.NON_HA);
+        underTest.setParameters(request, SdxDatabaseAvailabilityType.NON_HA, null);
 
         verify(request).setAws(captor.capture());
         AwsDatabaseServerV4Parameters awsDatabaseServerV4Parameters = captor.getValue();
         assertEquals("false", awsDatabaseServerV4Parameters.getMultiAZ());
         assertEquals(0, awsDatabaseServerV4Parameters.getBackupRetentionPeriod());
     }
 
+    @Test
+    public void testEngineVersion() {
+        underTest.setParameters(request, SdxDatabaseAvailabilityType.NON_HA, null);
```

Comment:
`null`? Should not it be `13`?

Reply:
I would guess we have a default value for provider if it is null